sdcm/fill_db_data.py

```diff
@@ -2972,7 +2979,10 @@ def fill_db_data(self):
             session.default_consistency_level = ConsistencyLevel.QUORUM
             session.set_keyspace("keyspace_fill_db_data")
             # clean original test data by truncate
-            self.truncate_tables(session)
+            try:
```

Comment:
not sure this general try-except is needed. perhaps it's good enough without it.

Reply:
If the last retry fails, exception will raise and break the subtest, that's not expected.